.github/workflows/main/limitCacheSize.py

```diff
@@ -0,0 +1,30 @@
+import os
+import collections
+
+# 400 megabytes
+sizeLimit = 400 * 1024 * 1024
+
+CacheEntry = collections.namedtuple( "CacheEntry", [ "file", "size", "mtime" ] )
+
+totalSize = 0
+cacheEntries = []
+
+for root, dirs, files in os.walk( "./sconsCache" ) :
```

Comment:
What do you think about parameterising this, then the only place that decides where `./sconsCache` is `main.yml`?

Reply:
This is the exact same script we used before just fine in the Travis setup. I don't see any real benefit in parameterising something which we always intend to be a constant.